frontend/pages/post/[id].js

```diff
@@ -184,6 +186,94 @@ const PostArticle = props => {
   )
 }
 
+// POST CONTENT COMPONENT
+class Comments extends React.Component {
+  static contextType = AppContext
+  state = {
+    comment: '',
+    errors: [],
+    comments: this.props.comments,
+    authenticatedUser: this.props.authenticatedUser
+  }
+  componentWillReceiveProps({ comments, authenticatedUser }) {
```

Comment:
last time I remember, this is already deprecated. There is only 3 things remaining:

- ComponentDidMount
- ComponentDidUpdate
- ComponentWillUnMount

Reply:
I replaced it with static getDerivedStateFromProps and that works.